UI/installer/mp-installer.nsi

```diff
@@ -55,14 +55,21 @@ RequestExecutionLevel admin
 ; Modern interface settings
 !include "MUI.nsh"
 
-!define MUI_ICON "obs.ico"
+!define MUI_ICON "..\..\cmake\winrc\obs-studio.ico"
 !define MUI_HEADERIMAGE_BITMAP "OBSHeader.bmp"
 !define MUI_WELCOMEFINISHPAGE_BITMAP "OBSBanner.bmp"
 
 !define MUI_ABORTWARNING
+!define MUI_FINISHPAGE_TITLE "Completed Setup"
 !define MUI_FINISHPAGE_RUN
-!define MUI_FINISHPAGE_RUN_TEXT "Launch OBS Studio ${SHORTVERSION}"
+!define MUI_FINISHPAGE_RUN_TEXT "Launch ${APPNAMEANDVERSION}"
 !define MUI_FINISHPAGE_RUN_FUNCTION "LaunchOBS"
+!define MUI_FINISHPAGE_SHOWREADME "https://github.com/obsproject/obs-studio/releases/tag/${APPVERSION}"
+!define MUI_FINISHPAGE_SHOWREADME_TEXT "View Release Notes"
+!define MUI_FINISHPAGE_SHOWREADME_NOTCHECKED
+!define MUI_FINISHPAGE_LINK "New to OBS? Check out our 4 step Quickstart guide"
```

Comment:
Nitpick:  End with punctuation.

I think, anyway.  I'm leaning towards ending with punctuation because you use punctuation for the first sentence.